lldb/source/Utility/Status.cpp

```diff
@@ -109,6 +109,13 @@ llvm::Error Status::ToError() const {
 
 Status::~Status() = default;
 
+const Status &Status::operator=(Status &&other) {
+  m_code = other.m_code;
+  m_type = other.m_type;
+  m_string = other.m_string;
```

Comment:
`std::move(other.m_string)` ?

Reply:
Sure, but this is going to be deleted in the next commit anyway.